Sources/SWBCLibc/libc.c

```diff
@@ -18,3 +18,16 @@ int swb_clibc_anchor(void);
 int swb_clibc_anchor(void) {
     return 0;
 }
+
+#include <stdio.h>
+
+extern FILE * const swb_stdout();
+extern FILE * const swb_stderr();
+
+FILE * const swb_stdout() {
```

Comment:
I believe this and `swb_stderr()` below need to be marked with `__declspec(dllexport)` on Windows.